WcaOnRails/app/views/registrations/register.html.erb

```diff
@@ -72,7 +72,7 @@
 
         <% if @competition.base_entry_fee %>
           <%= alert :info do %>
-            <%= t 'registrations.entry_fee' %><strong><%= humanized_money_with_symbol @competition.base_entry_fee %></strong>
+            <%= t 'registrations.entry_fee' %> <strong><%= humanized_money_with_symbol @competition.base_entry_fee %></strong>
```

Comment:
As per my comment below, I think this would be better as:

```erb
<%= t 'registrations.entry_fee', fee: humanized_money_with_symbol @competition.base_entry_fee %>
```

This loses the `<strong>...</strong>` tags, but I think that's okay. If we really want to keep them, we could make this a `*_html` key, which you can see examples of elsewhere.